docker-compose.mysql.yml

```diff
@@ -0,0 +1,11 @@
+---
+version: "3.7"
+services:
```

Comment:
I recommend you add an `environment:` override here for the `NAUTOBOT_DB_ENGINE` see https://github.com/nautobot/nautobot/pull/1514 for more details

Reply:
Done